core/ulog/README.md

```diff
@@ -42,3 +46,25 @@ logging:
   stdout: true
   level: debug
 ```
+
+User can initialize their own `zap.Logger` implementation and inject into ulog.
```

Comment:
Are you guys alright with using "you" instead of "user," since these docs are for the user?

If so…
- User --> You
- their --> your
- ulog --> ``ulog``